.github/workflows/markdown-linter.yml

```
@@ -0,0 +1,58 @@
+name: markdown-linter
+
+on:
+  pull_request:
+    branches: [master]
+    paths:
+      - "blog/**.md"
```

```suggestion
      - "blog/_posts/**.md"
```
Should this be in the posts subdirectory?

---

we could limit it to the `_posts` directory but markdown files can be in other directories like `_pages`.